ElectronicObserver/Window/Wpf/ExpeditionCheck/ExpeditionCheckViewModel.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using ElectronicObserver.Behaviors.PersistentColumns;
+using ElectronicObserver.Data;
+using ElectronicObserver.Observer;
+using ElectronicObserver.Resource;
+using ElectronicObserver.ViewModels;
+using ElectronicObserver.ViewModels.Translations;
+
+namespace ElectronicObserver.Window.Wpf.ExpeditionCheck;
+
+public class ExpeditionCheckViewModel : AnchorableViewModel
+{
+	public ExpeditionCheckTranslationViewModel ExpeditionCheckTranslation { get; }
+
+	public ObservableCollection<ExpeditionCheckRow> Rows { get; } = new();
+
+	public List<ColumnProperties> ColumnProperties { get; set; } = new();
+	public List<SortDescription> SortDescriptions { get; set; } = new();
+
+	public ExpeditionCheckViewModel() : base(Ioc.Default.GetService<ExpeditionCheckTranslationViewModel>()!.Title, "ExpeditionCheck",
+		ImageSourceIcons.GetIcon(IconContent.FormExpeditionCheck))
+	{
+		ExpeditionCheckTranslation = Ioc.Default.GetService<ExpeditionCheckTranslationViewModel>()!;
+
+		LoadData();
+		SubscribeToApis();
+	}
+
+	private void SubscribeToApis()
+	{
+		Utility.Configuration.Instance.ConfigurationChanged += LoadData;
+
+		APIObserver o = APIObserver.Instance;
+
+		o.ApiReqHensei_Change.RequestReceived += Updated;
+		o.ApiReqKousyou_DestroyShip.RequestReceived += Updated;
+		o.ApiReqKaisou_Remodeling.RequestReceived += Updated;
+
+		o.ApiPort_Port.ResponseReceived += Updated;
+		o.ApiGetMember_Ship2.ResponseReceived += Updated;
+		o.ApiReqKousyou_DestroyShip.ResponseReceived += Updated;
+		o.ApiGetMember_Ship3.ResponseReceived += Updated;
+		o.ApiReqKaisou_PowerUp.ResponseReceived += Updated;
+		o.ApiGetMember_SlotItem.ResponseReceived += Updated;
+		o.ApiReqHensei_PresetSelect.ResponseReceived += Updated;
+		o.ApiReqKaisou_SlotExchangeIndex.ResponseReceived += Updated;
+		o.ApiReqKaisou_SlotDeprive.ResponseReceived += Updated;
+		o.ApiReqKaisou_Marriage.ResponseReceived += Updated;
+	}
+
+	private void Updated(string apiname, dynamic data)
+	{
+		LoadData();
+	}
+
+	private void LoadData()
+	{
+		var db = KCDatabase.Instance;
+		Rows.Clear();
+
+		List<ExpeditionCheckRow> rows = db.Mission.Values.Select(mission => new ExpeditionCheckRow()
```

Comment:
```cs
.OrderBy(m => m.MapAreaID)
.ThenBy(m => m.ID)
```
Order the expedition before doing the select so that expedition check has the correct order by default.